skills/declarative/whoSkill/whoSkill/dialogs/FindPersonDialog/FindPersonDialog.dialog

```diff
@@ -23,7 +23,8 @@
           },
           "resultProperty": "$Result",
           "token": "=user.token.token",
-          "nameToSearchFor": "=$nameEntity"
+          "nameToSearchFor": "=$nameEntity",
+          "maxCount": 16
```

Comment:
16 feels like an odd number here... Just want to make sure this is right

Reply:
Yeah it is. The idea here is that if we get more than 15 then we just also return the "There are too many matches, please specify scenario". We don't need to get the exact count on the UI, just knowing it's got more than 15 is enough.